jme3-core/src/main/java/com/jme3/util/TestableExecutor.java

```diff
@@ -122,6 +123,15 @@ public void launch(String javaPackage, Object userData, String[] signatures) thr
         }
     }
 
+    /**
+     * Retrieves the current running {@link Testable}.
+     *
+     * @return the instance of the current running testable
+     */
+    public Testable getCurrentActiveTestable() {
+        return currentActiveTestable;
+    }
```

Comment:
This could be useful if you want to setup a Key action for example that will exit the current state test or the current app test (after testing) and then the utility takes over and starts the new queued test automatically...

For example: 
```java
public void onAction(String key, boolean keyPressed, float tpf) {
        if (key.equals(Commands.EXIT_COMMAND.getKey())) {
              final Testable activeTestable = Testable.getInstance().getCurrentActiveTestable();
              if (activeTestable instanceof JmeAppTest) {
                     ((JmeAppTest) activeTestable).stop();
              } else {
                     getStateManager().detach(((JmeStateTest) activeTestable));
              }
}
```